### PR TITLE
Signup 404

### PR DESCRIPTION
**Pull request by dev-prabhat**
Added Signup and 404 page

### CODE REVIEW

**Review by srishtigautam21 — APPROVED** [review]
The is clean and understandable
Good to merge

**Review by mikesch803 — APPROVED** [review]
nice ui, good work,good to merge.

File: src/pages/Page404/Page404.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        <>
+          <main className="page-main">
+              <Header/>
+              <NavBar/>
+                <section className="page-content">
+                    <figure className="img__wrapper">
+                        <img 
+                            className="img-responsive" 
+                            src={process.env.PUBLIC_URL + "/image/page404.jpg"} 
+                            alt="page404"
```
Comment (amartya-rs) [inline src/pages/Page404/Page404.js:20]: suggestion: can use template literal

**Review by amartya-rs — APPROVED** [review]
Code looks fine to me. Good to merge with the suggestions.

**Review by Dev-AkshatGupta — APPROVED** [review]
Everything seems cool to me you can go ahead and merge it

**Pull request merged**